lightning/src/ln/channelmonitor.rs

```diff
@@ -1966,7 +1962,11 @@ impl<ChanSigner: ChannelKeys> ChannelMonitor<ChanSigner> {
 				}
 			}
 		}
-		self.onchain_tx_handler.block_connected(txn_matched, claimable_outpoints, height, &*broadcaster, &*fee_estimator, &*logger);
+
+		let broadcasted_commit_tx = self.onchain_tx_handler.block_connected(txn_matched, claimable_outpoints, height, &*broadcaster, &*fee_estimator, &*logger);
```

Comment:
Sorry, but this name makes it sound like block_connected returns a commitment tx that was broadcast, which had me scratching my head. I think renaming it to `did_broadcast_commitment_tx` would be much easier for readers. Or I'm just dense.